### PR TITLE
regression/karaoke: update for improved faux-italics

### PR DESCRIPTION
**Pull request by rcombs**
As per https://github.com/libass/libass/pull/676

### CODE REVIEW

**Comment by astiob** [comment]
FWIW I think @TheOneric’s established convention was to prefix all commits that change the tests (as opposed to the infrastructure) with `reg:`

**Comment by TheOneric** [comment]
> prefix all commits that change the tests (as opposed to the infrastructure) with reg:

yep, changes to regression test are usually prefixed by `reg:`, changes to crash tests with `crash:` *(though there were no changes to crash tests since squashing to the new master branch)*.

File: regression/run-all.sh [hunk 1]
```diff
@@ -27,10 +27,17 @@ find "$tstDir" -maxdepth 1 -mindepth 1 -type d ! -name ".*" -print0 \
...
+        elif [ ! -z "$SAVE" ]; then
+            mkdir -p "$1/out"
+            OUTARGS=("-o" "$1/out")
+        fi
         if [ -f "$1"/scale ] ; then
-            "$2" "$1" -s "$(cat "$1"/scale)" -p "$ART_REG_TOLERANCE"
+            "$2" "$1" -s "$(cat "$1"/scale)" -p "$ART_REG_TOLERANCE" ${OUTARGS[*]}
         else
-            "$2" "$1" -p "$ART_REG_TOLERANCE"
+            "$2" "$1" -p "$ART_REG_TOLERANCE" ${OUTARGS[*]}
```
Comment (TheOneric) [inline regression/run-all.sh]: This seems unrelated to the new tests. It also uses bash-specific arrays making it nonportable. *(By not utilising `"${OUTARGS[@]}"` expansion preserving members with spaces, it's actually no better than a plain string variable later expanded without quotes)*

**Review by TheOneric — CHANGES_REQUESTED** [review]

File: regression/run-all.sh (hunk 1 above)
Comment (rcombs) [reply regression/run-all.sh]: Whoooops, didn't mean to include that in this PR.

**Review by TheOneric — APPROVED** [review]

**Pull request merged**